### ISSUE TITLE
sp_Blitz: Check 152: WAIT_XTP_OFFLINE_CKPT_NEW_LOG is not filtered out

### ISSUE THREAD

**Issue by rpellicaan** [BrentOzarULTD/SQL-Server-First-Responder-Kit#460]
Bug: This rule gives a false positive on wait type WAIT_XTP_OFFLINE_CKPT_NEW_LOG. 

According to Paul Randall (https://www.sqlskills.com/help/waits/wait_xtp_offline_ckpt_new_log/) this is a benign wait type.

Consider filtering out the wait types below too, as Paul does in his script. (http://www.sqlskills.com/blogs/paul/wait-statistics-or-please-tell-me-where-it-hurts/)

('BROKER_EVENTHANDLER'),
('CHKPT'),

('EXECSYNC'),
('FSAGENT'),

('KSOURCE_WAKEUP'),
('MEMORY_ALLOCATION_EXT'),
('PREEMPTIVE_XE_GETTARGETSTATE'),
('PWAIT_ALL_COMPONENTS_INITIALIZED'),
('PWAIT_DIRECTLOGCONSUMER_GETNEXT'),
('QDS_ASYNC_QUEUE'),
('REQUEST_FOR_DEADLOCK_SEARCH'),
('RESOURCE_QUEUE'),
('SERVER_IDLE_CHECK'),
('SLEEP_BPOOL_FLUSH'),
('SLEEP_DBSTARTUP'),
('SLEEP_DCOMSTARTUP'),
('SLEEP_MASTERDBREADY'),
('SLEEP_MASTERMDREADY'),
('SLEEP_MASTERUPGRADED'),
('SLEEP_MSDBSTARTUP'),
('SLEEP_TEMPDBSTARTUP'),
('SNI_HTTP_ACCEPT'),
('SQLTRACE_WAIT_ENTRIES'),
('WAIT_FOR_RESULTS'),
('WAITFOR_TASKSHUTDOWN'),
('WAIT_XTP_RECOVERY'),
('WAIT_XTP_HOST_WAIT'),
('WAIT_XTP_OFFLINE_CKPT_NEW_LOG'),
('WAIT_XTP_CKPT_CLOSE'),
('XE_DISPATCHER_JOIN')

**Comment by BrentOzar** [comment]
Hi Rob. Thanks for the tip! I went ahead and added that new top wait you suggested, but I wasn't sure what the remaining list was for - if that was a bug in your eyes or not.

Going forward, for relatively simple bugs like this (adding things to a list), feel free to go ahead and contribute code yourself. Here's how to get started:

https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/blob/master/CONTRIBUTING.md

Thanks!

**Comment by mjswart-d2l** [comment]
FWIW, I don't consider this one benign. I'll blog about it and return here. 
TLDR, Checkpoints can take a long time when in-memory tables are enabled.

**Comment by BrentOzar** [comment]
I'M WAITING SWART
I'M WAITING

**Comment by mjswart-d2l** [comment]
All right, hold your horses, it's scheduled to come out later this month.

**Comment by BrentOzar** [comment]
Every time I try to hold them, they kick me in the teeth.

**Comment by BrentOzar** [comment]
MICHAEL J SWART, YOUR HOMEWORK IS LATE

**Comment by mjswart** [comment]
What? How did it get to be October?

Honestly, That blog post I mentioned wasn't written or scheduled. I just assumed it would be.
I wanted to understand the behavior I am seeing. And thought that I'd get to the bottom of it by summer. 

Even after watching the https://groupby.org/conference-session-abstracts/in-memory-oltp-survival-guide/ I still have more questions than answers. I plan to bring some of those questions to the SQLCAT team at PASS, but it's not a given that at the end I'll have anything "bloggable". 

Long story short, I don't have anything interesting to say right now about the wait type: WAIT_XTP_OFFLINE_CKPT_NEW_LOG . 

I accept the incomplete mark on my homework but please don't tell my parents

**Comment by BrentOzar** [comment]
You're lucky you did so well in ice hockey class, buster. Now get out there and don't disappoint me on your midterm exams.

**Comment by mjswart** [comment]
Small follow up. I was being Chicken Little, or crying wolf. Pick your fable, I was wrong here. 

I was able to dig into a recurrence of an issue we had. Although our issue was XTP related and the wait WAIT_XTP_OFFLINE_CKPT_NEW_LOG shows up. Those two things are not directly related. I now consider this wait to be benign (like the rest of the world)

Sorry for the FUD
